examples/ConfigStoreDemo/ConfigStoreDemo.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
We have other sample apps on .net 7. Do you mind updating everything to .net 8?

Reply:
Yes missed it, will update